ynput/ayon-core
# OIIO Transcode: Simplify Extract OIIO transcode settings

## Changelog Description
This PR aims to ease and simplify the settings of `ExtractOIIOTranscode` plugin.

List of TODOs
- [x] Simplify the settings
- [x] Tweak the code to work with the new settings
- [x] Add Better Tool Tips
- [x] Add setting conversion


## Additional info
This PR was not requested by clients or community. 
The motivation came from documentation when this PR started initially as renaming the input fields.
but it evolved from continues discussions and testing to improve the user experience of the settings.

Therefore, we no longer need to explain why the display and view settings are shown if the transcode type is colorspace. 

This PR is not backward compatible as it changed the settings hierarchy a little bit.
But, conversion logic is implemented so we can copy old settings to the new settings without an issue.

For reference, I was testing BigRoy's PR [Collect required data for local and farm render jobs to have a 'colorspace' value specified #75](https://github.com/ynput/ayon-houdini/pull/75)




## Demo
![Animation_100](https://github.com/user-attachments/assets/406f6405-3a32-4ff8-ba36-61418da2e3b6)


## Testing notes:
1. Add your settings to the plugin. 
2. Verify it works as usual
